### DISCUSSION TITLE
Github Sponsor: Stripe account verified but not on github side

### DISCUSSION THREAD

**Discussion by dianedelallee** [community/community#22546] [New to GitHub]
<p>I followed all the step to create a sponsor for my github account.</p>
<p>I created my stripe account and everything seems fine on their side:  I get a confirmation from their support.</p>
<p>Unfortunately, on github side it tell me that my stripe account is unverified (in settings section), and in overview it tells me that I need to complete the required documentation, but I have already done all of that.</p>
<p>Does anyone can help me with this issue?</p>
<p>Thanks a lot</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi <a class="mention" href="/dianedelallee">@dianedelallee</a>! Have you tried contacting the <a href="https://support.github.com/contact/org-sponsors-waitlist">GitHub Sponsors team</a>? They would be the best to assist. I hope you’re able to resolve this soon :heart:</p>

**Comment by dianedelallee** [comment]
<p>Thanks for the link, this is what I was looking for, but can not find it.</p>
<p>Thanks a lot for your help <a class="mention" href="/liligalante">@liligalante</a></p>

**Comment by dianedelallee** [comment]
<p>Thanks again,<br>
the Github sponsor team solved the issue !</p>
<p>Really appreciate your help :pray:t3:</p>

**Comment by queenofcorgis** [comment]
<p>So glad I could help :slight_smile: And even more glad you got a solve!</p>
